Sources_v3/StreamChatUI/ChatChannel/MessageComposer/MessageComposerDocumentAttachmentView.swift

```diff
@@ -0,0 +1,77 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class MessageComposerDocumentAttachmentView<ExtraData: ExtraDataTypes>: ChatMessageAttachmentInfoView<ExtraData> {
+    // MARK: - Properties
+    
+    public var discardButtonHandler: (() -> Void)?
+    
+    // MARK: - Subviews
+
+    public private(set) lazy var fileIconImageView: UIImageView = {
+        let imageView = UIImageView().withoutAutoresizingMaskConstraints
+        imageView.contentMode = .center
+        return imageView
+    }()
+
+    // MARK: - Overrides
+    
+    override open func setUp() {
+        super.setUp()
+        
+        let tapRecognizer = UITapGestureRecognizer(target: self, action: #selector(discard))
+        addGestureRecognizer(tapRecognizer)
+    }
+
+    override public func defaultAppearance() {
+        backgroundColor = .white
+        layer.cornerRadius = 15
+        layer.masksToBounds = true
+        layer.borderWidth = 1
+        layer.borderColor = uiConfig.colorPalette.incomingMessageBubbleBorder.cgColor
```

Comment:
I'd create extension `CALayer+MakeBorder.swift`, and create something for the corner radius as well,  maybe not now, but in future we can make some enum of corner radiuses or something so it is consistent across the app... But just think about this I'd say :)

Reply:
Yeah, I was also having this thought. 
Will add a ticket to our backlog for that cause not sure what is the best way handle it rn.